com.unity.render-pipelines.universal/Editor/Camera/CameraStackReorderableList.cs

```diff
@@ -0,0 +1,301 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.SceneManagement;
+using UnityEditorInternal;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
```

Comment:
is it necessary since you are writing in this namespace?

Reply:
the current namespace refeers to `UnityEditor` and I need the `UnityEngine`